web/components/TopBar/ResetButton.vue

```diff
@@ -89,9 +84,9 @@ export default defineComponent({
       snackbarstore.setServerInfoMessage(message);
     };
 
-    const reset = async () => {
+    const Reset = async () => {
```

Comment:
Rename it to `resetFn` (instead of `Reset`)

Reply:
@sanposhiho Thanks!